fabric-chaincode-shim/src/main/java/org/hyperledger/fabric/contract/annotation/Contract.java

```diff
@@ -38,4 +38,22 @@
      * @return Name of the contract to be used instead of the Classname
      */
     String name() default "";
+
+    /**
+     * Name of the LEDGER serializer that should be used with this contract
+     * 
+     * NOT CURRENTLY USED _ FUTURE
```

Comment:
On the off chance that it remains in the future forever, could you remove the ledger serializer please? (Sounds nice to have though!)